salesagility/SuiteCRM
# Fixed  Additional Details pop-up issues #146

- Issue with pop-ups overflow on list views in modules
- Issue with pop-ups overflow in calendar
## Description



Modification of positioning of pop-ups.


- Create some accounts records,
- Open listView,
- on record in end of line you see information icon "i",
- click on icon right-mouse,
  the pop-up window overflow RHS of screen,
  references issue #146
  
## Motivation and Context



Improve display of the pop-up window
## How To Test This


- Create some accounts records,
- Open listView,
- on record in end of line you see information icon "i",
- click on icon right-mouse,
- the pop-up window should appear such that way, that right top coner of the pop up window sgould be a few pixels bellow and moved a few pixels from left bottom corner of the icon.
- to make sure it is work you should repeat above steps for following modules: accounts, contacts, opportunities, leads, calls, meetings, tasks, notes, cases, targets and projects,
  ## Types of changes
  
  
### Final checklist
